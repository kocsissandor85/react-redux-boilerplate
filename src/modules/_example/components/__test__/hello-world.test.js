import React from "react";
import HelloWorld from "../hello-world";

describe("The hello world component", () => {

  it("renders", () => {
    const wrapper = mount(
      <HelloWorld/>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});
