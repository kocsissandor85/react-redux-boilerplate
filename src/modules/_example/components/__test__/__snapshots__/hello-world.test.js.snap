// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The hello world component renders 1`] = `
<_default>
  <p>
    Hello world!
  </p>
</_default>
`;
